### PR TITLE
[MTDSA-10014] - seperating v1 and v1r6 release6enabled values

### PR DESCRIPTION
**Pull request by DreamcastKnight**

### CODE REVIEW

File: it/support/V1IntegrationBaseSpec.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package support
+
+trait V1IntegrationBaseSpec extends IntegrationBaseSpec {
+  override val release6Enabled = false //sets release6Enabled to be false for all V1 integration tests
```
Comment (jeremystone) [inline it/support/V1IntegrationBaseSpec.scala:20]: No need for a comment where it just repeats what can be seen in the code.

File: it/support/V1R6IntegrationBaseSpec.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package support
+
+trait V1R6IntegrationBaseSpec extends IntegrationBaseSpec {
+  override val release6Enabled = true //sets release6Enabled to be true for all V1R6 integration tests
```
Comment (jeremystone) [inline it/support/V1R6IntegrationBaseSpec.scala:20]: No need for a comment where it just repeats what can be seen in the code.

File: it/v1/endpoints/AmendBenefitAmountsControllerISpecIntegrationBaseSpec.scala [hunk 3]
```diff
@@ -21,13 +21,12 @@ import play.api.http.HeaderNames.ACCEPT
 import play.api.http.Status._
 import play.api.libs.json.{JsObject, JsValue, Json}
 import play.api.libs.ws.{WSRequest, WSResponse}
-import support.IntegrationBaseSpec
+import support.V1IntegrationBaseSpec
 import v1.models.errors._
 import v1.stubs.{AuditStub, AuthStub, DesStub, MtdIdLookupStub}
 
-class AmendBenefitAmountsControllerISpec extends IntegrationBaseSpec {
+class AmendBenefitAmountsControllerISpecIntegrationBaseSpec extends V1IntegrationBaseSpec {
```
Comment (jeremystone) [inline it/v1/endpoints/AmendBenefitAmountsControllerISpecIntegrationBaseSpec.scala:28]: Don't think that (and other IT classes) should have been renamed. Leave as `AmendBenefitAmountsControllerISpec` etc  etc

Comment (DreamcastKnight) [reply it/v1/endpoints/AmendBenefitAmountsControllerISpecIntegrationBaseSpec.scala:28]: this was an accident, I didn't realise this had even been changed

**Pull request merged**